ee/dtr/admin/install/index.md

```diff
@@ -54,99 +58,68 @@ your profile on the left navigation pane, and select **Admin Settings > Docker T
-    skipping the need for entering your username and password.
+5. Verify that DTR is installed, either by navigating to `https://<ucp-fqdn>/manage/settings/dtr`, or by navigating in the UCP web UI to **Admin Settings > Docker Trusted Registry**.
 
-4. Verify that DTR is installed. Either:
-    *  See `https://<ucp-fqdn>/manage/settings/dtr`, or;
+    Under the hood, UCP modifies `/etc/docker/certs.d` for each host and adds DTR's CA certificate. UCP can then pull images from DTR because the Docker Engine for each node in the UCP swarm has been configured to trust DTR.
 
-    *  Navigate to **Admin Settings > Docker Trusted Registry** from the UCP web UI. Under the hood, UCP modifies `/etc/docker/certs.d` for each host and adds DTR's CA certificate. UCP can then pull images from DTR because the Docker Engine for each node in the UCP swarm has been configured to trust DTR.
+6. Reconfigure the load balancer back to the desired protocol and port.
 
-5. Reconfigure your load balancer back to your desired protocol and port.
 
+## 4. Confirm that DTR is Running
 
-## Step 4. Check that DTR is running
+1. Using a browser, navigate to the UCP web interface.
 
-1. In your browser, navigate to the UCP web interface.
+2. Select **Shared Resources > Stacks** in the left navigation pane. DTR should be listed as a stack.
 
-2. Select **Shared Resources > Stacks** from the left navigation pane. You should see DTR listed as a stack.
-
-3. To verify that DTR is accessible from the browser, enter your DTR IP address or FQDN on the address bar.
-Since [HSTS (HTTP Strict-Transport-Security)
-header](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security) is included in all API responses,
-make sure to specify the FQDN (Fully Qualified Domain Name) of your DTR prefixed with `https://`,
-or your browser may refuse to load the web interface.
+3. Verify that DTR is accessible from the browser by entering the DTR IP address or FQDN on the address bar. As an [HSTS (HTTP Strict-Transport-Security)
+header](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security) is included in all API responses, make sure to specify the FQDN (Fully Qualified Domain Name) of the DTR instance prefixed with `https://` or the web interface may fail to load.
 
     ![](../../images/create-repository-1.png){: .with-border}
 
 
-## Step 5. Configure DTR
-
-After installing DTR, you should configure:
+## 5. Configure DTR
 
-  * The certificates used for TLS communication. [Learn more](../configure/use-your-own-tls-certificates.md).
-  * The storage backend to store the Docker images. [Learn more](../configure/external-storage/index.md).
+Once DTR has been successfully installed, configure the certificates used for TLS communication ([learn more](../configure/use-your-own-tls-certificates.md)) and the storage backend in which to store the Docker images ([learn more](../configure/external-storage/index.md)).
 
```

Comment:
Remove "learn more" and use "TLS communication" and "store the Docker images" for the text.